README-UI.md

```diff
@@ -0,0 +1,183 @@
+# Centralized UI Workflows
+
+- [Methodology](#methodology)
+- [Versioning](#versioning)
+- [Using in your repository](#using-in-your-repository)
+- [Configuration](#configuration)
+  - [Node/Yarn configuration](#nodeyarn-configuration)
+  - [Linting](#linting)
+  - [Unit tests](#unit-tests)
+    - [Jest](#jest)
+    - [Bigtest](#bigtest)
+  - [Sonarcloud](#sonarcloud)
+  - [Translations](#translations)
+  - [Module descriptor generation](#module-descriptor-generation)
+  - [Module publication](#module-publication)
+- [Secrets](#secrets)
+
+To improve the consistency and reliability of the CI/CD workflows for the FOLIO front-end repositories, we are migrating to a centralized workflow. This enables more frequent and centralized updates, encourages shared standards, and reduces the maintenance burden on individual repositories.
```

Comment:
This PR is about migrating, but the description of _the work_ is separate from the description of the PR. How about, "These shared CI/CD workflows enable more frequent and centralized updates, encourage shared standards, and reduce the maintenance burden on individual repositories."